apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/ResourceNames.cs

```diff
@@ -127,7 +127,7 @@ public partial class CheckAndMutateRowRequest
         /// </summary>
         public TableName TableNameAsTableName
         {
-            get { return string.IsNullOrEmpty(TableName) ? null : Google.Cloud.Bigtable.V2.TableName.Parse(TableName); }
+            get { return string.IsNullOrEmpty(TableName) ? null : TableName.Parse(TableName); }
```

Comment:
This doesn't work because it will resolve TableName to the string property, not the type. It needs to be qualified somehow.